eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -90,4 +90,90 @@ function CompatibleConvertFrom-Yaml {
   else {
       return ConvertFrom-Yaml $content
   }
+}
+
+<#
+.SYNOPSIS
+Common function that will verify that the YmlFile being loaded exists, load the raw file and
+return the results of CompatibleConvertFrom-Yaml or report an exception and return null if
+there's a problem loading the yml file. The return is the PowerShell HashTable object.
+
+.DESCRIPTION
+Common function that will verify that the YmlFile being loaded exists, load the raw file and
+return the results of CompatibleConvertFrom-Yaml or report an exception and return null if
+there's a problem loading the yml file. This is just to save anyone needing to load yml from
+having to deal with checking the file's existence and ensure that the CompatibleConvertFrom-Yaml
+is made within a try/catch. The return is the PowerShell HashTable object from the
+CompatibleConvertFrom-Yaml call or $null if there was an issue with the convert.
+
+.PARAMETER YmlFile
+The full path of the yml file to load.
+
+.EXAMPLE
+CompatibleLoadAndConvertFrom-Yaml -YmlFile path/to/file.yml
+#>
+function LoadAndCompatibleConvertFrom-Yaml {
+  param(
+    [Parameter(Mandatory=$true)]
+    [string]$YmlFile
+  )
+  if (Test-Path -Path $YmlFile) {
+    try {
+      return Get-Content -Raw -Path $YmlFile | CompatibleConvertFrom-Yaml
+    }
+    catch {
+      Write-Host "CompatibleLoadAndConvertFrom-Yaml::Exception while parsing yml file $($YmlFile): $_"
+    }
+  }
+  else {
+    Write-Host "CompatibleLoadAndConvertFrom-Yaml::YmlFile '$YmlFile' does not exist."
+  }
+  return $null
+}
+
+<#
+.SYNOPSIS
+Given the Hashtable contents of a Yml file and an array of strings representing the keys
+return the value if it exist or null if it doesn't.
+
+.DESCRIPTION
+The Yaml file needs to be loaded via CompatibleConvertFrom-Yaml which returns the file as
+as hashtable. The Keys are basically the path in the yaml file whose value to return, or
+null if it doesn't exist. This function does is safely traverses the path, outputting an
```

Comment:
```suggestion
null if it doesn't exist. This function safely traverses the path, outputting an
```

Reply:
I'm making this change and will push it with the next commit. It's easier since I'm updating the file anyways.

Reply:
Description updated as per this suggestion with the 3rd commit.